tutor/src/helpers/task-plan.coffee

```diff
@@ -31,4 +32,16 @@ module.exports = {
   earliestDueDate: (plan) ->
     dates = _.map(plan?.tasking_plans, 'due_at')
     _.first(dates.sort()) or ''
+
+
+  apiEndpointOptions: (id, courseId) ->
+    task = TaskPlanStore.get(id)
```

Comment:
hm, no we need to append the ecosystem_id even for new tasks.  Since if they're cloned they should use the original ecosystem id, for now at least